### DISCUSSION TITLE
The aarch64 static release does not contain a static binary

### DISCUSSION THREAD

**Discussion by sundbp** [babashka/babashka#1584] [Q&A] [answered]
**version**

v1.3.181

**platform**

linux-aarch64

**problem**

The release files tagged as -static for this platform are in fact NOT static binaries.

**repro**

Just check `file bb` for the file inside the static release for linux aarch64.

**expected behavior**

Expected to be a static binary :)

**Comment by borkdude** [comment, marked as answer]
The aarch64 version is "mostly static" as GraalVM native-image calls it. It has everything _except glibc_. Only linux amd64 binaries can be fully static with GraalVM.
More info here:
https://www.graalvm.org/22.1/reference-manual/native-image/StaticImages/
